dashboard/templates/assignments.html

```diff
@@ -94,13 +87,11 @@ <h4>Assignments Due By Date</h4>
 </div>
 
```

Comment:
`<div ng-app="visApp" class="container" ng-controller="mainController">` we can add the `ng-app` directive to the `div` tag itself.